aws_piper/aws_piper.py

```diff
@@ -29,10 +50,12 @@ def deploy(config_file='config.yaml'):
 
     pipeline_id = create_response.get('pipelineId')
     parameter_objects = translator.definition_to_api_parameters(read_json_file(cfg.get('parameter_objects')))
-    parameter_values = translator.definition_to_parameter_values(read_json_file(cfg.get('parameter_values')))
     pipeline_definition = translator.definition_to_api_objects(read_json_file(cfg.get('pipeline_definition')))
+    parameter_values = get_parameter_values(cfg)
+
+    send_query_to_S3(cfg)
 
-    return client.put_pipeline_definition(pipeline_id, pipeline_definition, parameter_objects, parameter_values)
+    #return client.put_pipeline_definition(pipeline_id, pipeline_definition, parameter_objects, parameter_values)
```

Comment:
Why did you comment that ? How is the pipeline be uploaded to AWS ?

Reply:
ok, that's a standard mistake to be found in PR, I commented it only for tests and forgot to uncomment